src/main/scala/latis/ops/ToHapiTime.scala

```diff
@@ -25,62 +17,24 @@ import latis.util.LatisException
  */
 class ToHapiTime extends MapOperation {
```

Comment:
There was a lot of duplication here so I figured we could delegate to `FormatTime`. We only need that pesky `size` so we can write text data as binary to fulfill the HAPI spec for `bin` output.

I thought about extending it, but have learned that extension should be avoided.